### ISSUE TITLE
Get Nan while training

### ISSUE THREAD

**Issue by pin-yu** [lionelmessi6410/Neural-Networks-from-Scratch#2]
I get Nan when I run this command 
`python3 train.py --activation sigmoid --optimizer momentum --l_rate 4 --batch_size 1`

Using learning rate less than 1 will prevent this issue.
